grammars/scss.cson

```diff
@@ -291,27 +291,34 @@
   'at_rule_include':
     'patterns': [
       {
-        'begin': '\\s*((@)include\\b)\\s*'
+        'name': 'meta.at-rule.include.scss'
+        'comment': '@include with arguments'
+        'begin': '\\s*((@)include)\\s+([\\w-]+)\\s*\\('
         'captures':
           '1':
             'name': 'keyword.control.at-rule.include.scss'
           '2':
             'name': 'punctuation.definition.keyword.scss'
           '3':
             'name': 'entity.name.function.scss'
-          '4':
-            'name': 'entity.name.text.scss'
-        'end': '\\s*[\\)|\\s\\;]'
-        'name': 'meta.at-rule.include.scss'
+        'end': '\\)'
         'patterns': [
           {
             'include': '#function_attributes'
           }
-          {
-            'include': '#functions'
-          }
         ]
       }
+      {
+        'name': 'meta.at-rule.include.scss'
+        'match': '\\s*((@)include\\b)\\s*([\\w-]*)'
```

Comment:
Oh, this is done on purpose so a solitary `@include` will be matched as you can see in the specs